app/ui/legacy/src/main/res/values-bg/strings.xml

```diff
@@ -653,7 +653,7 @@ K-9 Mail е мощен, безплатен имейл клиент за Андр
   <string name="volume_navigation_title">Навигация с клавиш за звука</string>
   <string name="volume_navigation_message">В преглед на съобщението</string>
   <string name="volume_navigation_list">В списъка със съобщения</string>
-  <string name="hide_special_accounts_title">Скрий обща входяща кутия</string>
+  <string name="hide_unified_inbox_title">Скрий обща входяща кутия</string>
```

Comment:
Renaming string keys is tricky. For translations we use [Transifex](https://www.transifex.com/k-9/k9mail). Changing the key of a string is equivalent to removing a string and adding a new one. So all translations for the old key on Transifex will be dropped and the new string will be initially untranslated.
The next time we pull translations from Transifex all changes in `values-*/strings.xml` will be overwritten. I could pull in Translations from Transifex, update with your changes, and then push to Transifex and hope there haven't been any changes in the meantime. But I generally try to avoid that.

I think it's best to live with the outdated name for now. Maybe add a TODO comment to the main strings file stating that the key should be renamed. Once we have enough cases like this we can do a mass rename, giving prior notice to translators.